core/iwasm/aot/aot_loader.c

```diff
@@ -2262,9 +2277,14 @@ load_import_funcs(const uint8 **p_buf, const uint8 *buf_end, AOTModule *module,
                     return false;
                 }
             }
-            linked_func = aot_loader_resolve_function(
-                module_name, field_name, declare_func_type, error_buf,
-                error_buf_size);
+            if (!sub_module)
```

Comment:
IMU, there is a case:

Let's say there is 2 imports.
- 1st import is not in a built-in. `sub_module` will be assigned in L2273.
- 2st import is a built-in. It will pass `if` block in L2272~L2279. Because `sub_module` has been assigned before, it goes to L2285 to search imported in `sub_module` when it should be found in as a built-in.

may `sub_module=NULL` after using.

Reply:
Yes, it's a bug, I added `sub_module=NULL`.